librecores-ci-openrisc/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM librecores/librecores-ci:0.3.0
+FROM librecores/librecores-ci:0.4.0
```

Comment:
not going to build properly after merge. Downside of the monorepo as we discussed before